packages/rainbowkit/src/css/sprinkles.css.ts

```diff
@@ -136,6 +136,8 @@ const responsiveProperties = defineProperties({
   },
   defaultCondition: 'smallScreen',
   properties: {
+    overflowY: ['auto', 'scroll', 'hidden'],
+    flexWrap: ['wrap', 'nowrap'],
```

Comment:
Adding these so we can have better responsive UI like this:
<img width="476" alt="Screenshot 2023-09-27 at 21 24 09" src="https://github.com/rainbow-me/rainbowkit/assets/53529533/705022ee-bcd2-4f99-8483-133f62c41113">